### DISCUSSION TITLE
install 24.09

### DISCUSSION THREAD

**Discussion by watchmax3** [o3de/o3de#18382] [Help Wanted]
another version another dying to run program: i have last version cmake and python with environment variables path of them... but and o3de get to %100 , ( first cmd pop up and try download python now i have python even that cmd didn't pop up ) said (msi) package didn't install. error  0x80070643. what i do? change windows? change living country? what is going on again? thanks and thanks very very much!

**Comment by vault-thirteen** [comment]
Today O3DE is a heavily-bugged engine. I tried the latest installer 24.09.1 and was shocked that a default empty project can not even compile. It looks like this engine has very low quality of testing. If you look at the milestones for the `Release/2305 Stabilization` target ( https://github.com/o3de/o3de/milestones ) in GitHub, you will see that it is only 47% complete. It looks like project management is very strange here. I hope that they improve as soon as possible.

**Reply by Andre-LA** [reply]
Unfortunately some bugs only happens on certain computers, for instance on my windows (11) PC a default template did opened and it did exported the game launcher when I tried.

**Reply by vault-thirteen** [reply]
You are lucky. The strangest thing is that the compilation process breaks due to some weird error with Python, not the engine itself. I want to see the latest version of the engine, but I am not expert in Python, so I do not know how to fix all this.

**Reply by watchmax3** [reply]
go to manu engine. need no programming and have own market. o3de is cooked by big tech companies . they didn't focus on o3de community and o3de is now just a toy for some nerd in university...

**Reply by guillaume-haerinck** [reply]
I understand your frustration and thanks for taking the time to create [an issue](https://github.com/o3de/o3de/issues/18523) to report the problem (outlined a solution in my last awnser). I completely agree that the install/build process could be greatly improved, but there are a few things that I want to highlight to put more context into the why and how's

- O3DE cannot distribute legally some of its dependencies itself, which is why in the case of python dependencies we have to rely on the python package manager which adds a layer of possible issues, which only occurs on some computers
- O3DE is in a very different area than Godot and other open sourced engine as that it is first and foremost a AAA shaped engine with all of the bloat that this imply. It is a huge codebase that is easy to extend for any game company, but for small team/solo developer it means longer build times and less streamlined setup
- It is a free project and free forever, while there are still bumps that needs to be worked on, the engine already improved a lot in terms of usuability and setup, there is a build pipeline to check that each new change is not breaking the build or some features, and I only see these set of tests improving as the time go

**Reply by vault-thirteen** [reply]
The list of features supported by the engine is really impressive. It just needs some more testing and stability.

**Reply by ShalokShalom** [reply]
What exactly is it, that prevents the legal distribution of Python?
